ChessVariantsLogic/Rules/Predicates/ChessPredicates/ForEvery.cs

```diff
@@ -22,14 +22,15 @@ public ForEvery(IPredicate rule, Player player)
     /// <param name="_">Irrelevant since this method looks at every possible future board, not just one</param>
     /// <param name="thisBoard">The current board state</param>
     /// <returns>True if the internal rule holds for every possible move in the current board state for the internal player, otherwise false.</returns>
```

Comment:
Outdated
```suggestion
```

Reply:
```c#
/// <summary>
/// Evaluates to true if the internal rule holds for every possible move in the nextState of the supplied boardTransition state for the internal player, otherwise false.
/// </summary>
/// <param name="transition">The board transition to be evaluated.</param>
/// <returns>True if the internal rule holds for every possible move in the nextState of the supplied boardTransition state for the internal player, otherwise false.</returns>
```
Updated documentation.